e2e-tests/portal/cypress/integration/journeys/maker/facilities/fixtures/AIN-deal-submitted-3-months-more.js

```diff
@@ -0,0 +1,93 @@
+const deal = require('./MIA-deal-with-accepted-status-issued-facilities-cover-start-date-in-past');
+const CONSTANTS = require('../../../../../fixtures/constants');
+
+const fourMonthsPast = () => {
+  const date = new Date();
+  date.setMonth(date.getMonth() - 4);
+  return date.valueOf();
+};
+
+const AINDeal = {
+  ...deal,
+  status: CONSTANTS.DEALS.DEAL_STATUS.UKEF_ACKNOWLEDGED,
+  previousStatus: CONSTANTS.DEALS.DEAL_STATUS.READY_FOR_APPROVAL,
+  submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.AIN,
+  details: {
```

Comment:
can we override these properties separately?
i.e.
```js
const ainDeal = deal;

deal.status = ....
...
deal.details.createdDate = fourMonthsPast();
deal.mockFacilities[0].createdDate = fourMonthsPast();
```

and so on. The reason being, a lot of the details are replicated from one file to another. Someone needs to maintain those

Reply:
done